gptables/examples/penguins_theme.py

```diff
@@ -9,8 +9,7 @@
 Where you wish to provide no metadata in required parameters, use ``None``.
 
 The theme parameter must take either a directory or a yaml file in the ``gptables.write_workbook`` function. 
-The yaml file used in this example can be found in the themes folder as ''penguins_test_theme.yaml''.
-The personalised theme removes any bold or italics from the table.
+The yaml file used in this example can be found in the themes folder as ''gptheme.yaml''.
```

Comment:
I think it is worth keeping a separate theme for this example, because I'd like to avoid people editing the default theme (both for good practice and because it's actually protected and might confuse people)